imports/server/mediasoup.ts

```diff
@@ -122,9 +122,7 @@ class Watchdog extends EventEmitter {
 
   resetTimeout() {
     this.lastActivity = new Date();
-    if (this.timeoutHandle) {
-      clearTimeout(this.timeoutHandle);
-    }
+    Meteor.clearTimeout(this.timeoutHandle);
```

Comment:
Are we intentionally dropping the `if (this.timeoutHandle) {` guard?

Reply:
It was never necessary - `timeoutHandle` is created in the constructor and is never set to undefined.